command/server.go

```diff
@@ -371,6 +371,19 @@ func (c *ServerCommand) Run(args []string) int {
 	// Initialize the listeners
 	lns := make([]net.Listener, 0, len(config.Listeners))
 	for i, lnConfig := range config.Listeners {
+		if lnConfig.Type == "atlas" {
+			cluster, err := core.Cluster()
```

Comment:
When the `server` command is being run, Vault will not be unsealed (since it is still coming up!). When Vault isn't unsealed, Cluster() won't return anything, it might actually error out.

@evanphx Did this work already?